tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -40,6 +40,13 @@
 from .sampler import Sampler, SampleState, SampleStateTensors
 from .scheduler import RequestScheduler, ScheduledRequests
 
+torch._C._activate_gpu_trace()
+torch.cuda._gpu_trace.register_callback_for_event_synchronization(lambda event: logger.info(f"TorchEvent {event} synchronized"))
```

Ruff (0.12.2)

Line too long (128 > 120)

(E501)